CMakeLists.txt

```diff
@@ -202,7 +248,10 @@ add_library(freertos_kernel STATIC
 target_include_directories(freertos_kernel
     PUBLIC
         include
-        ${FREERTOS_CONFIG_FILE_DIRECTORY}
 )
 
-target_link_libraries(freertos_kernel freertos_kernel_port)
+target_link_libraries(freertos_kernel
+    PUBLIC
+        freertos_config
```

Comment:
IMO, this should be left to the user.

Reply:
@paulbartell  - Agree - Yes - it is left to the user - just as it was left to the user to define `FREERTOS_CONFIG_FILE_DIRECTORY`.